### ISSUE TITLE
[DOC] Line length for code samples

### ISSUE THREAD

**Issue by karellm** [cibernox/ember-power-select#289]
Just went through the website docs. Looks like you've built an impressive component here! I will finally be able to remove Ember.Select, selectize and xSelect (yes... we have the 3 of them).

As I was reading the docs, I think there is a huge readability improvement you could do:

![](https://s3.amazonaws.com/f.cl.ly/items/0f1g2c163g3p2S463O1z/Image%202016-02-04%20at%204.10.22%20p.%20m..png?v=269c7aa1)

In our application we follow the 80 character / line rule with a "you might go slightly over but dont you dare testing us after 120" clause. The main reason is quite simple, horizontal scrolls suck (for us it is in github during code reviews). Here is the style we adopted:

```hbs
# bad :(
{{#power-select options=countries searchField="name" selected=selected onchange=(action (mut selected)) as |country|}}
  {{country.name}}
{{/power-select}}

# good :)
{{#power-select
  options=countries
  searchField="name"
  selected=selected
  onchange=(action (mut selected))
  as |country|
}}
  {{country.name}}
{{/power-select}}
```

It looked meh to me at first, but now there is no going back. Code reads and reviews A LOT better. I didn't see a repo for the site but if there is and you want a PR, let me know.

**Comment by cibernox** [comment]
The repo for the component and the repo for the docs is the same. The documentation is the dummy app :smiley: 

I'd also like to improve code samples with syntax highlighting and line numbers, but I hand't had time yet, I've been preparing the component for FastBoot lately.

Any PR that improves the docs in any way is certainly welcomed.

**Comment by karellm** [comment]
@cibernox Do you agree with the style above? If so I will work on a PR.

**Comment by cibernox** [comment]
Yes, basically break a line if you can't see fully it in a laptop screen without scrolling horizontally. 
I don't expect people to consult the documentation in a phone.

**Comment by karellm** [comment]
Fixed by #290
